src/ParkContainer/ParkContainer.tsx

```diff
@@ -39,26 +39,30 @@ const ParkContainer: React.FC<ParkContainerProps> = props => {
 
   const addToVisited = (parkCode: string) => {
     const addedPark = findChosenPark(parkCode)
-
+    addedPark.visited = true
```

Comment:
Brilliant little fix here that fixed 2 things! Nice!

Reply:
❤️ two birds one stone